src/main/java/uk/ac/ebi/ampt2d/accession/study/StudyConfiguration.java

```diff
@@ -19,27 +19,57 @@
 package uk.ac.ebi.ampt2d.accession.study;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import uk.ac.ebi.ampt2d.accession.ApplicationConstants;
 import uk.ac.ebi.ampt2d.accession.study.persistence.StudyAccessioningDatabaseService;
 import uk.ac.ebi.ampt2d.accession.study.persistence.StudyAccessioningRepository;
+import uk.ac.ebi.ampt2d.commons.accession.autoconfigure.EnableSpringDataContiguousIdService;
+import uk.ac.ebi.ampt2d.commons.accession.generators.DecoratedAccessionGenerator;
+import uk.ac.ebi.ampt2d.commons.accession.generators.monotonic.MonotonicAccessionGenerator;
+import uk.ac.ebi.ampt2d.commons.accession.persistence.monotonic.service.ContiguousIdBlockService;
 
 @Configuration
 @ConditionalOnProperty(name = "services", havingValue = "study-accession")
```

Comment:
We won't want to have the beans on conditional properties anymore, we want to deploy it as a single application.